torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2612,6 +2612,58 @@ def generator():
 
     return list(generator())
 
+
+def wrapper_cudnn_disabled(op):
+
+    def wrapped(*args, **kwargs):
+        with torch.backends.cudnn.flags(enabled=True):
+            return op(*args, **kwargs)
+
+    return wrapped
```

Comment:
Last thing before I merge this: let's remove the cudnn wrapper because OpInfos should probably not be in the business of also testing without cudnn (that seems backend-specific and I'm sure we have separate tests for that)